feedback-module/src/assets/constants.js

```diff
@@ -45,3 +45,12 @@ export const SCREEN1_BUTTONS = [
   { text: "Something appears to be broken or inaccurate", data: BROKEN },
   { text: "I have other feedback on this page or website", data: OTHER },
 ];
+
+export const SUBMISSION_TEXT = {
+  submitted: "Thank you for your feedback! It has been submitted.",
```

Comment:
We might want to change the variable names of "submitted" and "confidential" just because they are pretty specific to this text so if it changes in the future we might have some problems